src/schema/v2/homeView/zones/default.ts

```diff
@@ -57,6 +61,25 @@ const SECTIONS: HomeViewSection[] = [
  * Assemble the list of sections that can be displayed
  */
 export async function getSections(context: ResolverContext) {
+  const isMixerEnabled = isFeatureFlagEnabled("onyx_enable-home-view-mixer")
+
+  if (isMixerEnabled) {
+    return await getSectionsViaMixer(context)
+  } else {
+    return await getSectionsLegacy(context)
+  }
+}
+
+async function getSectionsViaMixer(context: ResolverContext) {
+  const mixer = new HomeViewMixer([
```

Comment:
I'm really into this DSL! Except - the 'filter' lingo - the boosting hero units one doesn't actually 'filter' - it more reorders.

And then the displayable one - it's feeling middleware-ish - and I'd imagine order specified in the mixer _may_ even  matter.

I wonder if instead of settling on the base 'SectionFilter' naming - instead maybe something like 'Step' or 'Rule' or 'Policy'?

Reply:
Yeah I took inspiration from (god help me) Elasticsearch and its notion of "token filters", which have the ability to reorder, modify, drop or even add tokens.

But I could roll with another name. `Rule` sounds nice and neutral.

Reply:
(it just feels like 'rule' or 'policy' is a touch more accurate and up front with intention than 'filter'. For that same reason, _maybe_ even instead of mixer something like 'composer'?)

Reply:
I agree with Matt that Filter might not be the best lingo here. Maybe Ranker, since these classes basically re-rank sections?

Reply:
I think "ranker" downplays the potential of these filters to remove or add sections as well.

So, based on that and some [further discussion](https://artsy.slack.com/archives/C05EQL4R5N0/p1745595474594619?thread_ts=1745488800.435159&cid=C05EQL4R5N0) I've decided to split the difference and keep "mixer" but change "filter" to "rule" per Matt's suggestion above.